src/pretalx.example.cfg

```diff
@@ -9,14 +9,16 @@ static = /var/pretalx/static
 debug = False
 url = https://eventyay.mydomain.com
 name = eventyay
+# To select a variant from CALL_FOR_SPEAKER_LOGIN_BTN_LABELS.
+call_for_speaker_login_button_label = default
 
 [database]
 backend = postgresql
 name = pretalx
 user = pretalx
 # For PostgreSQL on the same host, we don't need one because we can use
 # peer authentification if our PostgreSQL user matches our unix user.
-password = Change this in production or you're going to have a bad time
+password = Change this in production or you"'"re going to have a bad time
```

Comment:
Why does this need to be changed?

Reply:
To make it follow proper "ini" syntax, otherwise it breaks some viewer.
![image](https://github.com/user-attachments/assets/92686ec0-3243-4ac8-a4fe-37a0c0522d6d)